_posts/2021-10-5-packages-in-roblox.md

```diff
@@ -0,0 +1,139 @@
+So in this new script we've created in the `Red` part, we're going to add a simple piece code.
+
+```lua
+local Part = script.Parent
+local Light = Part.SurfaceLight
+
+Part.Color = Light.Color
+```
+*simple, right?*
+
+Go ahead and press `Run` (or F8) and check that the code works, if not, check the instructions below. Then copy the script and paste it into the other two parts, Green and Blue. Press `Run` again to ensure all the **SurfaceLights** match up with the part's color.
+
+Then, in the `Red` part, open the script again and add these changes.
+
+```lua
+local Part = script.Parent
+local Light = Part.SurfaceLight
+
+Part.Color = Light.Color
+Part.Material = Enum.Material.Neon
+Part.Size = Vector3.new(5, 5, 5)
+```
+
+Just like before, press `Run` and ensure the code works and after that copy the modified script and overwrite the previous scripts inside of the other two parts.
+
+### Creating a Package
+Now that we've gotten the easy part out of way, you may begin to see how tedious this process could be, manually updating every script and keeping track of them. That's why packages will come in handy when these type of situations happen as you develop your game.
+
+So let's create our first package! *(keep in mind that packages save into your inventory)*
+1. Right-click the script located in the **Red** part. *(or either of the three, just know that it'll be the script you'll create changes inside of)*
+2. Click `Convert to Package...` from the menu.
+    - As you do that, a dialog window will appear. Fill out the **Title** to `Test Package` or anything you'd like.
+    - Press **Submit**
+3. Once you've submitted, you should see something called **PackageLink** inside the script. That's a way to know you've successfully created your own package!
+
+### Accessing Your Packages and Inserting Them
+There are two places where you can access your packages, [`Toolbox`](https://developer.roblox.com/en-us/resources/studio/Toolbox) or the **Packages** folder located in [`Asset Manager`](https://developer.roblox.com/en-us/resources/studio/Asset-Manager).
+
+Now for our convienience, we'll be accessing our packages via **Asset Manager**. If you don't know where it is, right-click on the ribbon (if not there, Ctrl + F1), and click on **Asset Manager**.
+
+Access the new package inside of the **Packages** folder and you should be able to see the package that you just created. From here, delete the Scripts in the other two parts where that doesn't have a **PackageLink** inside of them and then insert the package into each part.
+
+### Committing Changes to Packages
+It doesn't matter which package you edit from as long as it's the same as the others. Now that we have multiple packages, let's figure out how to commit changes to all of the packages in the game. *(you can also update packages in other games as long they're the same and the permissions are met)*
+
+First, let's add some revisions to our code. Open the package and edit the code to this.
```

Comment:
Add some new line boundaries around code blocks, this is necessary in order to render it correctly